### PR TITLE
Update Getting Started Guide for Gutenberg 17.2

### PR DESCRIPTION
**Pull request by jonathanbossenger**
## What?

Updates the create-block command to use the @latest dist-tag, which matches the create-block documentation. 

## Why?

This helps if folks have an older version of create-block cached locally, which doesn't include the newer templating and changes to scaffolding, like automatically setting the render property in block.json

## How?

Updating the command in the Interactivity API Getting Started guide

It's a documentation change, so no suggested testing instructions or screenshots.

### CODE REVIEW

**Review by juanmaguitar — APPROVED** [review]

**Comment by juanmaguitar** [comment]
LGTM ✅

**Comment by jonathanbossenger** [comment]
@juanmaguitar I added some more updates to this getting started guide after the 17.2 release, in order to get a plugin working and testable.

File: packages/interactivity/docs/1-getting-started.md [hunk 1]
```diff
@@ -23,26 +23,38 @@ To get started with the Interactivity API, you can follow this [**Quick Start Gu
 We can scaffold a WordPress plugin that registers an interactive block (using the Interactivity API) by using a [template](https://www.npmjs.com/package/@wordpress/create-block-interactive-template) with the `@wordpress/create-block` command.
 
 ```
-npx @wordpress/create-block my-first-interactive-block --template @wordpress/create-block-interactive-template
+npx @wordpress/create-block@latest my-first-interactive-block --template @wordpress/create-block-interactive-template
 ```
 
-#### 2. Generate the build 
+> **Note**
```
Comment (juanmaguitar) [inline packages/interactivity/docs/1-getting-started.md:29]: I'm OK with adding this clarification as long as we remove it as soon as this issue is handled by the [scripts package](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/) and no additional action is required from the user.

I would feel better if before merging these changes we create an issue to remove this part when conditions allow (and share a link to it in this PR as a comment).

I'm approving these changes nevertheless

**Review by juanmaguitar — APPROVED** [review]

File: packages/interactivity/docs/1-getting-started.md (hunk 1 above)
Comment (jonathanbossenger) [reply packages/interactivity/docs/1-getting-started.md:29]: Agreed, I'll create that issue and keep an eye on it.

Comment (jonathanbossenger) [reply packages/interactivity/docs/1-getting-started.md:29]: @juanmaguitar [issue created](https://github.com/WordPress/gutenberg/issues/56936). 

Please feel free to assign it to me and I'll keep an eye on when the note can be removed.

**Comment by luisherranz** [comment]
The fix included in this PR is not working for me.

If I add the `"files": [ "src/view.js" ]` entry in the `package.json`, I only get these files in the zip:

```
Creating archive for `my-first-interactive-block` plugin... 🎁

Using the `files` field from `package.json` to detect files:

  Adding `build/index.js`.
  Adding `src/view.js`.
  Adding `package.json`.
  Adding `readme.txt`.

Done. `my-first-interactive-block.zip` is ready!
```

If I remove that entry from the `package.json`, I get these files in the zip:

```
Creating archive for `my-first-interactive-block` plugin... 🎁

Using Plugin Handbook best practices to discover files:

  Adding `my-first-interactive-block.php`.
  Adding `readme.txt`.
  Adding `build/block.json`.
  Adding `build/index.asset.php`.
  Adding `build/index.css`.
  Adding `build/index.css.map`.
  Adding `build/index.js`.
  Adding `build/index.js.map`.
  Adding `build/render.php`.
  Adding `build/style-index.css`.
  Adding `build/style-index.css.map`.

Done. `my-first-interactive-block.zip` is ready!
```

**Comment by luisherranz** [comment]
Using `"files": [ "[^.]*" ]` adds all files, which I think it's better:

```
Creating archive for `my-first-interactive-block` plugin... 🎁

Using the `files` field from `package.json` to detect files:

  Adding `build/index.css`.
  Adding `build/style-index.css`.
  Adding `src/edit.js`.
  Adding `build/index.js`.
  Adding `src/index.js`.
  Adding `src/view.js`.
  Adding `build/block.json`.
  Adding `src/block.json`.
  Adding `package.json`.
  Adding `build/index.css.map`.
  Adding `build/index.js.map`.
  Adding `build/style-index.css.map`.
  Adding `src/README.md`.
  Adding `build/index.asset.php`.
  Adding `my-first-interactive-block.php`.
  Adding `build/render.php`.
  Adding `src/render.php`.
  Adding `src/editor.scss`.
  Adding `src/style.scss`.
  Adding `readme.txt`.
  Adding `my-first-interactive-block.zip`.

Done. `my-first-interactive-block.zip` is ready!
```

**Pull request merged**